### ISSUE TITLE
gulp-util@3.0.8 is deprecated

### ISSUE THREAD

**Issue by t202wes** [JeffreyWay/laravel-mix#1581]
I'm re-creating this issue, because this problem continues to happen, yet the issues are incorrectly marked as closed.  Either re-open the old issue, and close this one, or keep this one open with the others closed.

```
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5

  npm WARN prefer global node-gyp@3.6.2 should be installed with -g
```

https://github.com/JeffreyWay/laravel-mix/issues/1401#issuecomment-373482138
https://github.com/JeffreyWay/laravel-mix/issues/1401

**Comment by JeffreyWay** [comment]
What specifically do you want me to do to fix that issue?

**Comment by ankurk91** [comment]
Ref https://github.com/imagemin/gifsicle-bin/issues/84

**Comment by wdj-ac** [comment]
Getting same issue with today installation:
```
yarn add v1.5.1
info No lockfile found.
[1/4] Resolving packages...
warning laravel-mix > img-loader > imagemin-gifsicle > gifsicle > bin-build > download > gulp-decompress > gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
[2/4] Fetching packages...
info fsevents@1.1.3: The platform "linux" is incompatible with this module.
info "fsevents@1.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Saved lockfile.
success Saved 749 new dependencies.
```

**Comment by ankurk91** [comment]
This is just a WARNING message, this is very normal in npm ecosystem. And `node_modules` folder has nothing to do with Laravel on a production server.
Laravel mix is supposed to run on  a CI server.

**Comment by OlivierVanEester** [comment]
Ref. https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5

> ...While not as urgent, the plugin author should still transition away from gulp-util.

**Comment by SodomousPrime** [comment]
Does anyone have an idea how to fix this?

**Comment by JeffBeltran** [comment]
Since I was tired of seeing this warning and this issue comes up when I google it, I figured I’d do a dive and see what is actually happening…

Laravel Mix isn’t the problem, it’s with one of it’s dependencies farther down the line. Right now the hold up is with the `gifsicle` dependency as it currently requires an older version of `bin-build` that still references the older version of `download`. One of the contributors of `gifsicle` commented that they want/need to update their dependency of `bin-wrapper` before they will tag a release that would fix this error (https://github.com/imagemin/gifsicle-bin/issues/90). So unfortunately, this warning isn't something a Mix maintainer/contributor/user can fix themselves.

**TLDR;** Not an issue with Mix and you can't fix it. Just ignore the warning as it doesn’t break anything.

**Comment by ronpx** [comment]
It's 2020. Any solution to this? 

`npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines `

**Comment by patybn3** [comment]
it mean that the version you have is deprecated, run 
`npm ls gulp-util`
to see what are the plug ins that are affecting your gulp-util.
mine was gulp-sass which version was 3.1.0. I updated the gulp-sass version and the error disappeared.
read this article:
https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
